code.swift

```diff
@@ -54,25 +54,24 @@ class PeriodicTableViewController: UIViewController {
     }
     
     // This next method was added by a new member of the team. Identify the 10 formatting, naming, and logic issues below:
-    @IBAction func chooseRandomElement(_ sender: Any){
-        let ranElem = elementsContoller.elements.randomElement()
+    @IBAction func chooseRandomElement(_ sender: Any) {
+        let ranElem = elementsContoller.elements
         
-        var chosenElementView:ElementView!
```

Comment:
You found the issue here, but the other change you made actually causes the code to not compile any more... can you guess why?